Guess game/Program.cs

```diff
@@ -10,282 +10,98 @@ namespace Guess_game
-                    return First;
-                }else if (GuessCount == 1 && guess == "agurkas")
-                {
-                    Console.WriteLine("Nice you get it ");
-                    First = First + Progress;
-                    Console.Write("Press Enter to continue ");
-                    Console.ReadLine();
-                    return First;
-                }else if (GuessCount == OutOfGuess)
-                {
-                    Console.WriteLine("You did not get it ");
-                    Progress = Progress - 3;
-                    First = First + Progress;
-                    Console.Write("Press Enter to continue ");
-                    Console.ReadLine();
-                    return First;
-                }
             }
-
-            return First;
-
+            return Progress;
         }
-        static int SecondRiddle(int Second)
-        {
-            int GuessCount = 0;
-            int OutOfGuess = 3;
-            int Progress = 3;
-            string guess = "";
 
-            Console.WriteLine("be ranku be koju duris atidaro");
-
-           
-
-            while (guess != "vejas")
-            {
-                Console.Write("Enter your guess ");
-                guess = Console.ReadLine();
-
-                GuessCount++;
-
-                if (GuessCount == OutOfGuess && guess == "vejas")
-                {
-                    Console.WriteLine("Nice you get it ");
-                    Progress = Progress - 2;
-                    Second = Second + Progress;
-                    Console.Write("Press Enter to continue ");
-                    Console.ReadLine();
-                    return Second;
-                }
-                else if (GuessCount == 2 && guess == "vejas")
-                {
-                    Console.WriteLine("Nice you get it ");
-                    Progress = Progress - 1;
-                    Second = Second + Progress;
-                    Console.Write("Press Enter to continue ");
-                    Console.ReadLine();
-                    return Second;
-                }
-                else if (GuessCount == 1 && guess == "vejas")
-                {
-                    Console.WriteLine("Nice you get it ");
-                    Second = Second + Progress;
-                    Console.Write("Press Enter to continue ");
-                    Console.ReadLine();
-                    return Second;
-                }
-                else if (GuessCount == OutOfGuess)
-                {
-                    Console.WriteLine("You did not get it ");
-                    Progress = Progress - 3;
-                    Second = Second + Progress;
-                    Console.Write("Press Enter to continue ");
-                    Console.ReadLine();
-                    return Second;
-                }
-            }
-
-            return Second;
-
-        }
-        static int ThirdRiddle(int Third)
+        static void Intruduction()
```

Comment:
methods must be named as Verbs not nouns (you can read about it in the google: https://www.dofactory.com/reference/csharp-coding-standards)

Reply:
Also this method name contains syntax error